variables.tf

```diff
@@ -451,6 +461,12 @@ variable "kiam_arn" {
   default     = ""
 }
 
+variable "oidc_provider" {
```

Comment:
Does this variable function as the toggle between using IRSA vs kiam?  I am assuming it does, but maybe a description in README.md or VARIABLES.md somewhere describing the two ways to do IAM in k8s would be good.

Reply:
added to Notes in README